### ISSUE TITLE
Bug in documentation 5.4??

### ISSUE THREAD

**Issue by qrczak** [laravel/docs#3018]
Maybe I'm wrong but in 5.4 documentation in Collection section in method **every()** [https://laravel.com/docs/5.4/collections#method-every](https://laravel.com/docs/5.4/collections#method-every) there is an example:
```
collect([1, 2, 3, 4])->first(function ($value, $key) {
    return $value > 2;
});

// false
```
But where is the every() method?

**Comment by Acen** [comment]
See c50acbb

**Comment by paulofreitas** [comment]
This was fixed by #3025. 👍

**Comment by laurencei** [comment]
Closing ticket as this has been fixed.
